tests/parametric/test_trace_sampling.py

```diff
@@ -344,3 +346,61 @@ def test_trace_dropped_by_trace_sampling_rule_tags(self, test_agent, test_librar
 
         assert span["metrics"].get(SAMPLING_PRIORITY_KEY) == -1
         assert span["metrics"].get(SAMPLING_RULE_PRIORITY_RATE) == 0.0
+
+
+@missing_feature(context.library == "python", reason="Not implemented")
+@missing_feature(context.library == "python_http", reason="Not implemented")
+@missing_feature(context.library == "java", reason="Not implemented")
+@missing_feature(context.library == "ruby", reason="Not implemented")
+@missing_feature(context.library == "nodejs", reason="Not implemented")
+@missing_feature(context.library == "php", reason="Not implemented")
+@missing_feature(context.library == "dotnet", reason="Not implemented")
+@missing_feature(context.library == "golang", reason="Not implemented")
```

Comment:
If we are using the decorator at class level, we should move it to manifest file